tests/xmltester/XMLTester.cpp

```diff
@@ -178,6 +178,17 @@ tolower(std::string& str)
         );
 }
 
+void
+toupper(std::string& str)
```

Comment:
Already implemented in `geos/util/string.h`

Reply:
great, thanks